var/spack/repos/builtin/packages/imagemagick/package.py

```diff
@@ -28,6 +28,7 @@ class Imagemagick(AutotoolsPackage):
     depends_on('ghostscript')
     depends_on('ghostscript-fonts')
     depends_on('libsm')
+    depends_on('pkg-config', type='build')
```

Comment:
```suggestion
    depends_on('pkgconfig', type='build')
```
This will use the virtual package